### DISCUSSION TITLE
Expresssion level of metabolizing enzymes in normal vs obese patients

### DISCUSSION THREAD

**Discussion by Sanjana2697** [Open-Systems-Pharmacology/Forum#947] [Questions & Problems]
Hi everyone,

I am working on obese vs normal patients data in PK-Sim. So I am facing an issue wherein PK-Sim I am not able to justify in PK-Sim the difference between normal and obese patients in terms of expression of metabolizing enzymes as in obese patients the drug is not able to pass to the plasma. The expression is relative to metabolizing enzymes i.e it is a gene database that selects the %expression in the tissue which will be the same in normal and obese. While in obese patients there has to be greater metabolism than in normal patients. How can we show that in PK-Sim?

Any suggestions?

Thank you,
Sanjana

**Comment by StephanSchaller** [comment]
Dear Sanjana,

I do hope I understand your issue correctly.
You could create two different baseline individuals with different expressions of the same enzyme (you can manually edit the loaded expression data) and then create your populations for each of these individuals and their distinct expression profiles.

Hope this helps,
Stephan

**Reply by Sanjana2697** [reply]
Hi Stephan,

Thank you so much for your help. Yes, I can edit the expression level manually in PK-Sim but how can we edit the expression level as it 1. relative to other tissues and 2. how can I calculate the percent expression of that metabolizing enzyme in fat tissue of the obese patients? 

Thank you
Sanjana

**Reply by jgg1822** [reply]
Hi Sanjana,

I've been working on some PBPK modeling with subjects with versus without obesity, and I'm curious what metabolic enzyme you're working with? Is it actually located/active in the fat tissue? I've found for my research that lean bodyweight organs (e.g. liver and kidney) are bigger in obese subjects, which can drive increased metabolism. It may be that you need to increase these organ volumes in virtual subjects with obesity to reflect the increased metabolism observed.

Thanks,
Jackie

**Reply by Sanjana2697** [reply]
Thank you for this information. 
Yes, the metabolic enzymes are present in the fat tissues. Aldo-keto reductase and carbonyl reductase are the enzymes I am working on within this model. As these enzymes are present in fats, I thought increasing their expression in fat tissues of obese patients will make some difference but there was no effect.
I can try with liver and kidney volume but I also want to see the difference in fats as well.
I don't know why it is not showing any change due to expression but if you have any idea of the action of the metabolizing enzyme in fats let me know, it would be a great help!

Thank you,
Sanjana

**Reply by prvmalik** [reply]
Hi Sanjana, 

Just confirming: The relative expression in fat/adipose should be manually changed in the Individual Building Block > Expression > Metabolic Enzymes. Ensure you are updating your individual from building block when running the simulation, and that the simulation is configured correctly.

To Jackie's point, this paper on Daunorubicin metabolism is good evidence that reductase activity is highest in liver, kidney and intestine, and present in low amounts in most other tissues. https://jpet.aspetjournals.org/content/jpet/177/3/567.full.pdf . Another PBPK model for doxorubicin has a similar inference (doi 10.1002/jps.10161).

**Reply by Sanjana2697** [reply]
Yes, I did in the exact same manner.
Okay, thank you so much. I will take a look at the papers.

Regards
Sanjana

**Reply by lau00070** [reply]
Hi everyone,

Currently, I'm also working on a PBPK project on obese patients. I know from Simcyp that there is a standard obese population in their settings based on the work described in PMID: 22087867. As far as I know, there is no current obese population set in PK SIM. Could that be created based on previous works, which probably makes it easier to further elaborate on?

Regards,
Cedric

**Reply by StephanSchaller** [reply]
Hi @lau00070 , the lab  from @aedginto is working on an obese population afaik. @TheiBa, do you know of any other work which could be built on (I think to remember your group worked on obesity some time ago?)?
